frontend/components/ClubPage/MemberCard.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import s from 'styled-components'
+import { BORDER } from '../../constants/colors'
+
+const Card = s.div`
+  text-align: center;
+  width: 24%;
+  box-sizing: border-box;
+  margin-right: 1%;
+  margin-bottom: 20px;
+  padding: 20px 10px;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  float: left;
```

Comment:
Ideally we don't use float since it can have odd behavior. Ideally `display: inline-block` will also get the job done